<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ApproveDeny</class>
 <widget class="QDialog" name="ApproveDeny">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>216</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButtonApprove">
      <property name="text">
       <string>Approve</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonDeny">
      <property name="text">
       <string>Deny</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>397</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLineEdit" name="lineEditName"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditPrice"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditPeriod"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
